plugin.audio.radiofrance/addon.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.audio.radiofrance" name="RadioFrance — Lives and Podcasts" version="1.0.0" provider-name="bateast">
+    <requires>
+        <import addon="script.module.requests" version="2.25.1" />
+        <import addon="xbmc.python" version="3.0.0" />
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en">https://www.radiofrance.fr</summary>
+        <license>GNU General Public License, v3</license>
```

Comment:
Please use the SPDX identifier: https://spdx.org/licenses/

Reply:
ok